src/components/Communicator/CommunicatorDialog/CommunicatorDialog.component.js

```diff
@@ -67,7 +69,9 @@ const CommunicatorDialog = ({
         <Tabs
           value={selectedTab}
           onChange={onTabChange}
-          className="CommunicatorDialog__tabs"
+          className={classNames('CommunicatorDialog__tabs', {
+            CommunicatorDialog__tabs__dark: dark
```

Comment:
Instead of passing a dark class to all sub components of `CommunicatorDialog` you could place just one class on the component's root `div`